webrtc.html

```diff
@@ -5945,10 +5945,24 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
       <div>
         <pre class="idl">dictionary RTCRtpCodecCapability {
              DOMString mimeType;
+             unsigned long  clockRate;
+             unsigned short channels = 1;
+             DOMString      sdpFmtpLine;
 };</pre>
         <section>
           <h2>Dictionary <dfn>RTCRtpCodecCapability</dfn>
           Members</h2>
+          <p>The <code>RTCRtpCodecCapability</code> dictionary provides
+          information about codec capabilities. Only capability
+          combinations that would utilize distinct payload types in a
+          generated SDP offer are provided.  For example:</p>
+          <ol>
+            <li>Exactly one Opus codec, with "stereo=1", indicating that
+              the browser supports either stereo or mono.</li>
```

Comment:
I think I was wrong about this example.

> Independent of the stereo parameter every receiver MUST be able to receive and decode stereo signals

So I suppose the "stereo" parameter in the capabilities would only represent the default preference of the browser. Maybe "max-fr" and "max-fs" with VP8 would be a better example.